source/trigger/cdc.go

```diff
@@ -0,0 +1,353 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trigger
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/conduitio/conduit-connector-postgres/common"
+	sdk "github.com/conduitio/conduit-connector-sdk"
+	"github.com/jackc/pgx/v4"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"go.uber.org/multierr"
+)
+
+const (
+	timeoutBeforeCloseDBSec        = 20
+	timeoutToClearTrackingTableSec = 5
```

Comment:
Why not make these constants a `time.Duration` type?

```suggestion
	timeoutBeforeCloseDB        = 20 * time.Second
	timeoutToClearTrackingTable = 5 * time.Second
```

Reply:
Yeah, it'll be much better 👍